src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return (Colors.values()[new Random().nextInt(
+                Colors.values().length)]).toString();
```

Comment:
Every invoke will make new object of Random, you need only one, what about field? In enum you can use name instead of toString